lib/carin_for_blue_button_test_kit/generated/v2.0.0/eob_group.rb

```diff
@@ -132,6 +135,9 @@ def self.metadata
       test from: :c4bb_v200_eob_pharmacy_must_support_test
       test from: :c4bb_v200_eob_professional_non_clinician_validation_test
       test from: :c4bb_v200_eob_professional_non_clinician_must_support_test
+      test from: :c4bb_v200_custom_eob_type_data_absent
+      test from: :c4bb_v200_custom_eob_insurer_same
+      test from: :c4bb_v200_custom_eob_outcome_complete
```

Comment:
If these were manually inputted here, they will disappear the next time the generator is run, so you will have to add code to the generator to add in these custom groups here!